packages/vip-go/services/generateLogInfo.js

```diff
@@ -1,9 +1,21 @@
 const { format } = require('util');
 
 module.exports = function generateLogInfo(method, messages) {
+  const hasUrl = messages.filter(
+    (message) => 'object' === typeof message && message.url
```

Comment:
could this be `messages.find` instead (resulting in a single value vs. an array)?